service/src/services/MeasureService.ts

```diff
@@ -1,5 +1,5 @@
 import { loggers, RequestArgs, RequestCtx } from '@projecttacoma/node-fhir-server-core';
-import { findResourceById, findResourcesWithQuery } from '../db/dbOperations';
+import { findDataRequirementsWithQuery, findResourceById, findResourcesWithQuery } from '../db/dbOperations';
```

Comment:
```suggestion
import { findDataRequirementsWithQuery, findResourceById, findResourcesWithQuery, createResource, updateResource } from '../db/dbOperations';
```
Same suggestion that was mentioned in LibraryService - can add the rest of the imports here